docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:22.04
```

Comment:
TF uses `20.04`. If we use a version that's too recent, its glibc may not work on older versions of TF. Does this work fine with `20.04`?